goerli.accounts.json

```diff
@@ -1 +1,9 @@
-{"605311267278208658008777842068112645483518374771058332871185239025837222499": {"address": "0x010365a34be3f137f663199fddd158ca41c1b2d774a3d7db36ddecaea9e7108e", "index": 0}}
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
@threepwave hey just in case you missed this, looks like the rebase conflict wasn't resolved for a few files?

Reply:
Thanks - haven't been getting much sleep the past few nights and missed this. Just resolved.

Reply:
haha no worries ser